.eslintrc

```diff
@@ -6,5 +6,9 @@
 	},
 	"rules": {
 		"@wordpress/i18n-text-domain": [ "error", { "allowedTextDomain": [ "wp-parsely" ] } ]
+	},
+	"globals": {
+		// Page variable for Puppeteer tests
```

Comment:
I know this isn't strictly JSON, but are comments are allowed here?

Reply:
Looks like it. The IDE didn't complain and ESlint picks the new configuration correctly.